docs/changes/units/17985.feature.rst

```diff
@@ -0,0 +1,2 @@
+Unit conversions between kelvins and degrees Rankine no longer require the
```

Comment:
```suggestion
Unit conversions between Kelvins and degrees Rankine no longer require the
```

Reply:
`kelvins` is correct. See section 5.3 in the [SI brochure](https://www.bipm.org/documents/20126/41483022/SI-Brochure-9-EN.pdf/2d2b50bf-f2b4-9661-f402-5f9d66e4b507?version=4.0) for the general principle and Table 2 for the kelvin in particular.